Annotation/alternative_golden_set.json

```diff
@@ -130,7 +130,7 @@
                     }
                 ],
                 "type": "alternative",
-                "clause": "",
+                "clause": "whether to engage his opponent or stand off",
                 "embedded clauses": []
             }
         ]

```

Comment:
I don't think it's an EC, unless you think of "based on" as the predicate and "what I've read" as the EC. But I think we should remove the second sentence so that each sentence in the dataset is a single sentence.